src/app.js

```diff
@@ -17,22 +21,28 @@ const app = express()
 app.use(express.json())
 app.use(expressLogger)
 
-app.get('/events', (req, res) => {
-  getEvents(req.query.location)
-    .on('error', (e) => {
-      logger.error(e)
-      res.status(500).end(e.message)
-    })
-    .pipe(res)
+app.get('/events', async (req, res) => {
```

Comment:
it's bad practice to declare endpoints directly in main file. Better to split them by related entities, create Routers and put in separate files and just include here. You can read more about `express.Router` here:
http://expressjs.com/en/guide/routing.html